### PR TITLE
Add debian folder to be able to do the debian package

### PR DESCRIPTION
**Pull request by circleng**
I've pushed the debian folder and files including changelog, compat, control, and rules.
And I also change the main.c and Makefiles to be compatible with the newer DPDK version (18-19). 
 
Here's a command to create a source package.
`sudo dpkg-buildpackage -uc -us --build=source`
The result will get the tarball "demu_(version).tar.gz" and "demu_(version).dsc"

After extract the source package. Here's a command to convert the source to the binary package.
`sudo debuild -uc -us`
The result will get the binary debian package "demu_(version)_(arch).deb" .

After installing the debian package, We can use 3 commands on /usr/bin
- demu-setup : The preparation part and usage part of DEMU. The input parameters are NICs name. The default parameter is enps010 and enps0f1 as shown on the GitHub page. These parameter also input the PCI number such as 01:00.0, 01:00.1, etc.
- demu-cleanup : This command restores the normal Linux network configuration.
- demu : This command calls the DEMU execution file at ./build/demu and able to use the same options from the this execution file.

### CODE REVIEW

**Comment by ryousei** [comment]
Thanks. 
* tell me why do you need demu wrapper script.
* add packaging section in README.md.

**Comment by ryousei** [comment]
I could build debian package. I think files under demu-build directory is not necessary.
`~# dpkg -c demu_0.1.0-1_amd64.deb 
drwxr-xr-x root/root         0 2020-03-04 06:50 ./
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/
-rwxr-xr-x root/root       112 2020-03-04 06:50 ./usr/bin/demu
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/demu-build/
-rwxr-xr-x root/root      1730 2020-03-04 06:50 ./usr/bin/demu-build/.demu.cmd
-rwxr-xr-x root/root       784 2020-03-04 06:50 ./usr/bin/demu-build/.main.o.cmd
-rwxr-xr-x root/root     13646 2020-03-04 06:50 ./usr/bin/demu-build/.main.o.d
-rwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/demu-build/_install
-rwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/demu-build/_postbuild
-rwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/demu-build/_postinstall
-rwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/demu-build/_preinstall
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/demu-build/app/
-rwxr-xr-x root/root     30776 2020-03-04 06:50 ./usr/bin/demu-build/app/demu
-rwxr-xr-x root/root    441729 2020-03-04 06:50 ./usr/bin/demu-build/app/demu.map
-rwxr-xr-x root/root     30776 2020-03-04 06:50 ./usr/bin/demu-build/demu
-rwxr-xr-x root/root    441729 2020-03-04 06:50 ./usr/bin/demu-build/demu.map
-rwxr-xr-x root/root     33856 2020-03-04 06:50 ./usr/bin/demu-build/main.o
-rwxr-xr-x root/root       550 2020-03-04 06:50 ./usr/bin/demu-cleanup
-rwxr-xr-x root/root       721 2020-03-04 06:50 ./usr/bin/demu-setup
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/share/
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/share/doc/
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/share/doc/demu/
-rw-r--r-- root/root       161 2020-03-04 06:50 ./usr/share/doc/demu/changelog.Debian.gz`

In short, the following is enough and better.
`drwxr-xr-x root/root         0 2020-03-04 06:50 ./
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/bin/
-rwxr-xr-x root/root       112 2020-03-04 06:50 ./usr/bin/demu
-rwxr-xr-x root/root       550 2020-03-04 06:50 ./usr/bin/demu-cleanup
-rwxr-xr-x root/root       721 2020-03-04 06:50 ./usr/bin/demu-setup
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/share/
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/share/doc/
drwxr-xr-x root/root         0 2020-03-04 06:50 ./usr/share/doc/demu/
-rw-r--r-- root/root       161 2020-03-04 06:50 ./usr/share/doc/demu/changelog.Debian.gz`

**Comment by circleng** [comment]
I've updated the packaging section on README.

I also remove the demu-build folder when installing the package and move the DEMU binary file to /usr/bin/demu to call this file directly. Here's the current list of files when installed.

```
/.
/usr
/usr/bin
/usr/bin/demu
/usr/bin/demu-cleanup
/usr/bin/demu-setup
/usr/share
/usr/share/doc
/usr/share/doc/demu
/usr/share/doc/demu/changelog.Debian.gz
```
For other scripts, The purpose of creating the demu-setup script is just for setup the DEMU faster after restart a server. I think it would be helpful, so I decided to include this file on the pull request.
